pkg/operator/ceph/disruption/clusterdisruption/pools.go

```diff
@@ -117,7 +117,7 @@ func (r *ReconcileClusterDisruption) reconcileCephObjectStore(cephObjectStoreLis
 
 		rgwCount := objectStore.Spec.Gateway.Instances
 		minAvailable := &intstr.IntOrString{IntVal: rgwCount - 1}
-		if minAvailable.IntVal <= 1 {
+		if minAvailable.IntVal < 1 {
 			break
```

Comment:
@sp98 @satoru-takeuchi  Should we `continue` here instead of `break`? It seems like we need a separate PDB for each object store, instead of breaking if one of them only has a single rgw.

Reply:
@travisn indeed, i'll fix it later.